Makefile

```diff
@@ -5,22 +5,24 @@
 
 help:
 	@echo "The following make targets are available:"
-	@echo "	 devenv		create venv and install all deps for dev env (assumes python3 cmd exists)"
-	@echo "	 dev 		install all deps for dev env (assumes venv is present)"
-	@echo "  docs		create pydocs for all relveant modules (assumes venv is present)"
-	@echo "	 package	package for pypi"
-	@echo "	 test		run all tests with coverage (assumes venv is present)"
-	@echo "	 testcore	run all tests excluding spark tests with coverage (assumes venv is present)"
-	@echo "	 testspark	run all tests of spark (assumes venv is present)"
+	@echo "  setupinpip	use pip to install requirements in current env"
+	@echo "	 setupinconda	use conda to install requirements in current env"
+	@echo "  docs 		create pydocs for all relveant modules (assumes venv is present)"
+	@echo "	 package 	package for pypi"
+	@echo "	 test 		run all tests with coverage (assumes venv is present)"
+	@echo "	 testcore 	run all tests excluding spark tests with coverage (assumes venv is present)"
+	@echo "	 testspark 	run all tests of spark (assumes venv is present)"
 	@echo "	 sql		fugue sql code gen"
 
 
-devenv:
+setupinpip:
```

Comment:
At least for now, for consistency, let's keep devenv and env and add setupinpip and setupinconda.
Later we can have a refactoring fugue project wise to replace devenv and dev for all repos if we decide that is the way to go